design-book/src/best-practices.md

```diff
@@ -0,0 +1,20 @@
+# Best Practices
+
+## Keep the UI calm
+
+- Avoid strong changes in colors, big jumps in layouts, or popping elements on simple interactions. For example strong mouse hover highlights can be very flashy and cause distracting visual noise; some subtle changes can be okay.
+
+- Avoid strong accent colors for no reason. Using accent colors when you aren't trying to get the user's attention only creates visual noise and clutter that harms the user experience. The UI should be calm with slight contrasts unless there's good reason to do so. Getting this right early on will do good for long term editor-user communication.
+
+- When ever Ui breaks away from Bevy Editor's design practices, the reasoning should be well thought and documented. Consistency is a double edge sword, it can just as easily harm usability as it helps. Breaking away can be an acceptable trade if there are good reasons to do so, and only then.
```

Comment:
```suggestion
- If the design breaks away from Bevy Editor's design practices, the reasoning should be well thought and documented. Consistency is a double edge sword, it can just as easily harm usability as it helps. Breaking away can be an acceptable choice if there are good reasons to do so, and only then.
```